src/hiero_sdk_python/query/transaction_record_query.py

```
@@ -102,37 +143,45 @@ def _should_retry(self, response: Any) -> _ExecutionState:
+            ResponseCode.PLATFORM_NOT_ACTIVE,
         }
-        
+
         if status == ResponseCode.OK:
-            if response.transactionGetRecord.header.responseType == query_header_pb2.ResponseType.COST_ANSWER:
+            if (
+                response.transactionGetRecord.header.responseType
+                == query_header_pb2.ResponseType.COST_ANSWER
+            ):
                 return _ExecutionState.FINISHED
             pass
```

Ruff (0.14.14)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)